### DISCUSSION TITLE
[Possible Issue]: DDIM sampler makes a messy console output

### DISCUSSION THREAD

**Discussion by brknsoul** [vladmandic/sdnext#1013] [General]
Using the DDIM sampler results in the console looking like;
![image](https://github.com/vladmandic/automatic/assets/1969381/767101a7-4a03-43ea-906d-91f04a14b90a)

I imagine that the DDIM maintainer would have to fix this?

**Comment by vladmandic** [comment]
progress bar gets messed up because of line being printed in-between - question is who is printing those lines:
> tensor (0.xxx device="privateuseone:0')

**Reply by brknsoul** [reply]
I imagine DDIM, since the numbers seem to be percentages.

**Reply by vladmandic** [reply]
its not DDIM, its something else that is printing those lines. i just did a quick search for string template and dont see it in core repo.
